packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -8839,18 +8839,29 @@ void main() {
     testWidgets(
       'iOS uses the system context menu by default if supported',
       (WidgetTester tester) async {
-        TestWidgetsFlutterBinding.instance.platformDispatcher.supportsShowingSystemContextMenu =
-            true;
-        _updateMediaQueryFromView(tester);
-        addTearDown(() {
-          TestWidgetsFlutterBinding.instance.platformDispatcher
-              .resetSupportsShowingSystemContextMenu();
-          _updateMediaQueryFromView(tester);
+        tester.platformDispatcher.supportsShowingSystemContextMenu = true;
+        addTearDown(() async {
+          tester.platformDispatcher.resetSupportsShowingSystemContextMenu();
+          // Pump one more root view to replace it with one that reads the
+          // updated reset value of supportsShowingSystemContextMenu for
+          // subsequent tests..
+          await tester.pumpWidget(
```

Comment:
I think we can use `tester.view.reset()` instead of pumping another root view. Found that here https://github.com/flutter/flutter/blob/5da19ccfbc17643fdd7e00615a369070d43e2055/packages/flutter/test/widgets/media_query_test.dart#L407